vllm/lora/models.py

```diff
@@ -776,24 +776,3 @@ def remove_oldest_lora(self) -> bool:
             self._registered_loras.remove_oldest()
             return True
         return False
-
-
-def create_lora_manager(
-        model: nn.Module,
-        max_num_seqs: int,
-        max_num_batched_tokens: int,
-        vocab_size: int,
-        lora_config: LoRAConfig,
-        lora_manager_cls: Type[LoRAModelManager] = LoRAModelManager,
-        **kwargs) -> LoRAModelManager:
-    """Create a LoRA adapter for a given model."""
-    if not hasattr(model, "supported_lora_modules"):
-        raise ValueError(f"Model {type(model)} is not supported for LoRA.")
```

Comment:
let's make sure this check is still present

Reply:
added, thanks for your reivew :)